web/src/message_notifications.js

```diff
@@ -43,16 +43,11 @@ function get_notification_content(message) {
         content = "New direct message from " + message.sender_full_name;
     }
 
-    if (content.length > 150) {
-        let i;
-        // Truncate content at a word boundary
-        for (i = 150; i > 0; i -= 1) {
-            if (content[i] === " ") {
-                break;
-            }
-        }
-        content = content.slice(0, i);
-        content += " [...]";
+    const characterLimit = 150;
+
+    if (content.length > characterLimit) {
+        // Truncate content not at a word boundary
+        content = content.slice(0, characterLimit - 3) + "...";
```

Comment:
Can we instead use the single-character unicode …? That would save 2 characters.

Reply:
Sure, we can use the single-character Unicode ellipsis (…) in it.

Reply:
I have completed the changes. Please take a moment to review the modifications made.